pkg/scraper/image.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/stashapp/stash/pkg/utils"
 )
 
-func setPerformerImage(ctx context.Context, client *http.Client, p models.ScrapedPerformer, globalConfig GlobalConfig) error {
+func setPerformerImage(ctx context.Context, client *http.Client, p *models.ScrapedPerformer, globalConfig GlobalConfig) error {
 	if p.Image == nil || !strings.HasPrefix(*p.Image, "http") {
```

Comment:
This (and in the other functions) used to have a nil check `p == nil || ...`, is it needed here again?

Reply:
It depends.

If you call the function from some random part of the code, then having a nil-check would be necessary. However, these functions are only called from a function where the value cannot be nil, which is the underlying reason to make the change in the first place.

A good way to follow up here is likely to get rid of the functions completely. The key operating function is `getImage(..)`, and if you hoist that into the caller, you avoid the pointered type altogether.